modules/apps/portlet-configuration/portlet-configuration-web/src/main/resources/META-INF/resources/edit_public_render_parameters.jsp

```diff
@@ -132,20 +132,22 @@ Set<PublicRenderParameter> publicRenderParameters = (Set<PublicRenderParameter>)
 	for (PublicRenderParameterConfiguration publicRenderParameterConfiguration : publicRenderParameterConfigurations) {
 	%>
 
-		var ignoreInput = document.getElementById(
-			'<portlet:namespace /><%= PublicRenderParameterConfiguration.IGNORE_PREFIX + HtmlUtil.escapeJS(publicRenderParameterConfiguration.getPublicRenderParameterName()) %>'
-		);
-		var mappingInput = document.getElementById(
-			'<portlet:namespace /><%= PublicRenderParameterConfiguration.MAPPING_PREFIX + HtmlUtil.escapeJS(publicRenderParameterConfiguration.getPublicRenderParameterName()) %>'
-		);
-
-		if (ignoreInput && mappingInput) {
-			mappingInput.disabled = ignoreInput.checked;
-
-			ignoreInput.addEventListener('click', () => {
-				Liferay.Util.toggleDisabled(mappingInput, !mappingInput.disabled);
-			});
-		}
+		(function () {
```

Comment:
Why this `(function(){ })()` block is added? I would said is not needed, could you please check by removing it? 🙏